.travis.yml

```diff
@@ -18,7 +18,7 @@ before_script:
 - fluxbox &
 - wget http://selenium-release.storage.googleapis.com/2.44/selenium-server-standalone-2.44.0.jar
   -O selenium-server-standalone.jar
-- java -jar selenium-server-standalone.jar > /dev/null 2>/dev/null &
+- java -jar selenium-server-standalone.jar > selenium.log &
```

Comment:
btw, 2> means "regular output", so that's why the second file is receiving that regular output
only if there is an error, it will be stored in the first file